### PR TITLE
feat(build image): update buildspec to work w/new pocket-build image

### PR DESCRIPTION
**Pull request by cmharlow**
## Goal

Follows successful review, merge & deploy of https://github.com/Pocket/shared-infrastructure/pull/506 & https://github.com/Pocket/shared-infrastructure/pull/507

Updates the repo's own Codebuild spec to work with the new pocket-build image, which doesn't require install steps (minus some uncertainty about perl-utils). Changes from alpine to ubuntu OS.

Blocked by [#507](https://github.com/Pocket/shared-infrastructure/pull/507) (which updates the codebuild project to use the new image which this buildspec will run on).

## Reference

Tickets:
* https://getpocket.atlassian.net/browse/INFRA-847

[INFRA-847]: https://getpocket.atlassian.net/browse/INFRA-847?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -45,15 +39,6 @@ phases:
...
-
-      # Until we use a docker image with tfenv built in lets install it.
-      # This lets us store the needed terraform verison in the source and not rely on changing amazon values.
-      - rm -rf /bin/terraform
-      - git clone https://github.com/tfutils/tfenv.git ~/.tfenv
-      - ln -s ~/.tfenv/bin/* /usr/bin
-      - tfenv install
-      - tfenv use $(cat .terraform-version)
-
       - npm install -g npm@latest
```
Comment (hyperparabolic) [inline buildspec.yml:42]: Is this setup being moved to `pocket/pocket-build:prod` image creation?

**Review by hyperparabolic — COMMENTED** [review]
Codebuild seems unhappy here around the npm install.  Is this line supposed to be removed?

File: buildspec.yml (hunk 1 above)
Comment (cmharlow) [reply buildspec.yml]: @hyperparabolic yes

**Review by hyperparabolic — APPROVED** [review]
👍

**Pull request merged**